src/Parse/Syntax/Parser.php

```diff
@@ -218,6 +218,13 @@ protected function evalTwigViewField($field, $params, $prefix = null)
             case 'checkbox':
                 $result = '{% if ' . $field . ' %}' . $params['_content'] . '{% endif %}';
                 break;
+            case 'datepicker':
+                if ($params['mode'] === 'date'){
```

Comment:
This doesn't actually properly support the `mode` option. The `mode` option supports three values, `date`, `time`, and `datetime`. Up to you if that makes more sense with a bunch of elseifs or a select case setup.